extensions/sql-migration/src/constants/strings.ts

```diff
@@ -828,14 +833,18 @@ export interface LookupTable<T> {
 }
 
 export const StatusLookup: LookupTable<string | undefined> = {
-	[MigrationStatus.InProgress]: localize('sql.migration.status.inprogress', 'In progress'),
-	[MigrationStatus.Succeeded]: localize('sql.migration.status.succeeded', 'Succeeded'),
-	[MigrationStatus.Creating]: localize('sql.migration.status.creating', 'Creating'),
-	[MigrationStatus.Completing]: localize('sql.migration.status.completing', 'Completing'),
-	[MigrationStatus.Retriable]: localize('sql.migration.status.retriable', 'Retriable'),
-	[MigrationStatus.Canceling]: localize('sql.migration.status.canceling', 'Canceling'),
-	[MigrationStatus.Canceled]: localize('sql.migration.status.canceled', 'Canceled'),
-	[MigrationStatus.Failed]: localize('sql.migration.status.failed', 'Failed'),
+	[MigrationState.Canceled]: localize('sql.migration.status.canceled', 'Canceled'),
+	[MigrationState.Canceling]: localize('sql.migration.status.canceling', 'Canceling'),
+	[MigrationState.Completing]: localize('sql.migration.status.completing', 'Completing'),
+	[MigrationState.Creating]: localize('sql.migration.status.creating', 'Creating'),
+	[MigrationState.Failed]: localize('sql.migration.status.failed', 'Failed'),
+	[MigrationState.InProgress]: localize('sql.migration.status.inprogress', 'In progress'),
+	[MigrationState.ReadyForCutover]: localize('sql.migration.status.readyforcutover', 'Ready for cutover'),
+	[MigrationState.Restoring]: localize('sql.migration.status.restoring', 'Restoring'),
+	[MigrationState.Retriable]: localize('sql.migration.status.retriable', 'Retriable'),
+	[MigrationState.Succeeded]: localize('sql.migration.status.succeeded', 'Succeeded'),
+	[MigrationState.UploadingFullBackup]: localize('sql.migration.status.uploadingfullbackup', 'Uploading full backup'),
+	[MigrationState.UploadingLogBackup]: localize('sql.migration.status.uploadinglogbackup', 'Uploading log backup'),
```

Comment:
NIT: can this be 'Uploading log backups'? More than likely a customer will have multiple

Reply:
I would suggest using "backup(s)' to be more general